### DISCUSSION TITLE
Avatar with status / nested avatars / status indicators / notification count

### DISCUSSION THREAD

**Discussion by jasonsturges** [radix-ui/themes#653] [Ideas]
I noticed the Radix UI Design System features avatars with status; or, nested avatars - any plan to incorporate that?

https://design-system.modulz-deploys.com/#avatar
<img width="537" alt="image" src="https://github.com/radix-ui/themes/assets/1213591/80c710ad-34e4-4b72-8e67-5eff804a12af">

That status component by itself would be super cool, as well:
https://design-system.modulz-deploys.com/#status
<img width="241" alt="image" src="https://github.com/radix-ui/themes/assets/1213591/2bd4241e-396d-4055-8709-c67046612dbd">

Material UI features this:
https://mui.com/material-ui/react-avatar/#with-badge
<img width="179" alt="image" src="https://github.com/radix-ui/themes/assets/1213591/84ad3e62-3aa0-4211-92ce-fddaf74f6d52">

Material UI has a different concept of "badges" that are super helpful:
https://mui.com/material-ui/react-badge/

Like mail alerts:
<img width="131" alt="image" src="https://github.com/radix-ui/themes/assets/1213591/65e7249b-f4a1-46bb-a51c-9c8866708499">

Shopping cart:
<img width="105" alt="image" src="https://github.com/radix-ui/themes/assets/1213591/2a479506-c27a-4092-b61f-eaaab857cd1d">

Implemented as:
<img width="297" alt="image" src="https://github.com/radix-ui/themes/assets/1213591/e783bfcb-b84c-4012-a01c-54c4c56b8843">

Or in general, just notification counts.

Maybe this is just an inset, but would be a nice feature for your consideration in Radix UI Themes.

**Comment by vladmoroz** [comment]
We might do that at some point, no immediate plans yet
